File: sandbox/node/src/BulkSendJobGetDefaultExample.ts
import * as fs from 'fs';
import api from "@dropbox/sign"
import models from "@dropbox/sign"

const apiCaller = new api.BulkSendJobApi();

const bulkSendJobId = "6e683bc0369ba3d5b6f43c2c22a8031dbf6bd174";
const page = 1;
const pageSize = 20;

apiCaller.bulkSendJobGet(
    bulkSendJobId,
    page,
    pageSize,
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling BulkSendJob#bulkSendJobGet:");
  console.log(error.body);
});

File: sandbox/ruby/src/BulkSendJobGetDefaultExample.rb
require "dropbox-sign"

Dropbox::Sign.configure do |config|
end

begin
    response = Dropbox::Sign::BulkSendJobApi.new.bulk_send_job_get(
        "6e683bc0369ba3d5b6f43c2c22a8031dbf6bd174",
        {
            page: 1,
            page_size: 20,
        },
    )

    p response
rescue Dropbox::Sign::ApiError => e
    puts "Exception when calling BulkSendJob#bulk_send_job_get: #{e}"
end
